camera_calibration/nodes/cameracalibrator.py

```diff
@@ -274,7 +276,7 @@ def buttons(self, display):
     def y(self, i):
         """Set up right-size images"""
         return 30 + 40 * i
-        
+
     def screendump(self, im):
         i = 0
         while os.access("/tmp/dump%d.png" % i, os.R_OK):
```

Comment:
how about the tmp here ? (not sure ... have you checked other mentions of the tmp path?)